src/wallet/mod.rs

```diff
@@ -1332,6 +1313,48 @@ where
         Ok(psbt)
     }
 
+    /// get the corresponding PSBT Input for a LocalUtxo
+    pub fn get_psbt_input(
+        &self,
+        utxo: LocalUtxo,
+        sighash_type: Option<SigHashType>,
+        force_non_witness_utxo: bool,
+    ) -> Result<Input, Error> {
+        // Try to find the prev_script in our db to figure out if this is internal or external,
+        // and the derivation index
+        let (keychain, child) = match self
+            .database
+            .borrow()
+            .get_path_from_script_pubkey(&utxo.txout.script_pubkey)?
+        {
+            Some(x) => x,
+            None => return Err(Error::UnknownUTXO),
+        };
```

Comment:
```suggestion
            .ok_or(Error::UnknownUTXO)?;
```

Reply:
Done. Thanks for the reminder. There is always just the right unpacking method lurking somewhere.